reana-ui/src/components/WorkflowActionsPopup.js

```diff
@@ -33,7 +33,7 @@ export default function WorkflowActionsPopup({ workflow, className }) {
   const [open, setOpen] = useState(false);
   const { id, size, status, session_status: sessionStatus } = workflow;
   const isDeleted = status === "deleted";
-  const isDeletedUsingWorkspace = isDeleted && size !== "0K";
+  const isDeletedUsingWorkspace = isDeleted && size.raw !== 0;
```

Comment:
I guess something like this is enough:

```suggestion
  const isDeletedUsingWorkspace = isDeleted && size.raw;
```